addon/components/polaris-button-group.js

```diff
@@ -1,6 +1,6 @@
 import Component from '@ember/component';
 import layout from '../templates/components/polaris-button-group';
-import wrapChildren from '../utils/wrap-children';
+import wrapChildren, { childrenWithoutClassName } from '../utils/wrap-children';
```

Comment:
@tomnez what do you think about renaming this util file to something like `dom` and making `wrapChildren` a named export instead of the default? Asking because `childrenWithoutClassName` doesn't really seem to belong in a util file with such a specific name as `wrap-children` IMO 🤔

Reply:
works for me